int32 id   #object id
float32 px  #position of x, meter
float32 py  #position of y, meter
float32 vx  #velocity of x, m/s
float32 vy  #velocity of y, m/s
uint32 type #0=unknow, 1=pedestrian, 2=bike, 3=car, 4=truck
